content/en/docs/contribute/suggesting-improvements.md

```diff
@@ -0,0 +1,66 @@
+---
+title: Suggesting content improvements
+slug: suggest-improvements
+content_template: templates/concept
+weight: 10
+card:
+  name: contribute
+  weight: 20
+---
+
+{{% capture overview %}}
+
+The easiest way to contribute to Kubernetes documentation file is to an issue about existing content,
+or a request for new content. All you need is a [GitHub account](https://github.com/join) and a web browser.
```

Comment:
Hello @celestehorgan , 
There are a lot of details to review -- specifically how the new topics and content fit together (with the
original sections) as well as the actual content changes.

Proposed sections:
```
Suggesting content improvements
Reviewing changes
-- | Reviewing pull requests
-- | For Approvers and Reviewers
<!-- Last original contribute section -->
Advanced Contributing
```